cns/requestcontroller/kubecontroller/crdrequestcontroller.go

```diff
@@ -0,0 +1,168 @@
+package kubecontroller
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	"github.com/Azure/azure-container-networking/cns/logger"
+	"github.com/Azure/azure-container-networking/cns/restserver"
+	nnc "github.com/Azure/azure-container-networking/nodenetworkconfig/api/v1alpha"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+const nodeNameEnvVar = "NODENAME"
+const k8sNamespace = "kube-system"
+const prometheusAddress = "0"
```

Comment:
maybe a comment here to indicate 0 means disabled (but I do like the name of the constant). I might look at this and think "`0` is pretty close to `0.0.0.0/0` so it must be binding to all addresses", which is the opposite of what it actually does